composer.json

```diff
@@ -16,25 +16,25 @@
         "drupal/antibot": "^1.3",
         "drupal/cached_computed_field": "^1.0-beta5",
         "drupal/cas": "^1.6",
-        "drupal/cas_account_link": "^1.0-alpha3",
+        "drupal/cas_account_link": "~2.0",
         "drupal/cas_attributes": "dev-1.x",
         "drupal/changed_fields": "^3.2",
         "drupal/config_exclude": "dev-1.x",
         "drupal/config_ignore": "dev-3.x",
         "drupal/config_readonly": "^1.0",
         "drupal/config_update": "dev-1.x",
-        "drupal/core": "^8.8.6",
-        "drupal/core-composer-scaffold": "^8.8.6",
+        "drupal/core": "~8.9",
+        "drupal/core-composer-scaffold": "~8.9",
         "drupal/diff": "~1.0",
-        "drupal/digital_size_formatter": "^1.0-alpha1",
+        "drupal/digital_size_formatter": "~2.0",
         "drupal/editor_file": "~1.4",
         "drupal/email_registration": "^1.0",
         "drupal/embed_block": "^1.0",
         "drupal/entity_legal": "^2",
         "drupal/error_page": "^1.0",
         "drupal/facets": "~1.4",
-        "drupal/field_group": "^1.0",
-        "drupal/file_url": "^1.0-alpha7",
+        "drupal/field_group": "^3.0",
```

Comment:
We cannot update `field_group` because the `3.x` the vertical tabs description are not displayed & labels CSS styling is different. We'll do this update in ISAICP-5835.